core/components/atoms/checkbox/checkbox.js

```diff
@@ -18,7 +18,12 @@ const Checkbox = props => (
   <Checkbox.Option
     readOnly={props.readOnly}
     {...Automation('checkbox')}
-    {...exclude('onChange', props)}
+    {
+      // Prevent passing the onChange handler
+      // both to the label and the checkbox.
+      // See: https://github.com/auth0/cosmos/pull/1473
+      ...exclude('onChange', props)
```

Comment:
👎 Still no clue what's going on 

> See: https://github.com/auth0/cosmos/pull/1473

lol @ recursion

Repeating myself:

> Just saw the `🚨hi priority` on this. We can merge this PR (I've tested that it works) and continue to improve the code.

Reply:
Commented it in the body of the PR. The recursion is intentional and is intended for future reference, as we've been doing since a couple of months.

Reply:
Oops sorry, didn't read the description. Honest mistake

Reply:
np, added it a couple minutes ago. It wasn't there before.